Library/Homebrew/extend/os/linux/cask/config.rb

```diff
@@ -0,0 +1,34 @@
+# typed: strict
+# frozen_string_literal: true
+
+require "os/linux"
+
+module OS
+  module Linux
+    module Cask
+      module Config
+        extend T::Helpers
+
+        requires_ancestor { ::Cask::Config }
+
+        module ClassMethods
+          DEFAULT_DIRS = T.let({
+            vst_plugindir:  "~/.vst",
+            vst3_plugindir: "~/.vst3",
+            fontdir:        "#{ENV.fetch("XDG_DATA_HOME", "~/.local/share")}/fonts",
+          }.freeze, T::Hash[Symbol, T.nilable(String)])
+
+          sig { returns(T::Hash[Symbol, T.untyped]) }
```

Comment:
We should be able to tighten up the types:
```suggestion
          }.freeze, T::Hash[Symbol, String])

          sig { returns(T::Hash[Symbol, String]) }
```